framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -568,7 +568,18 @@ qx.Class.define("qx.io.remote.Rpc",
             break;
 
           case 1: // async with handler function
-            handler(result, ex, id);
+            try
+            {
+              handler(result, ex, id);
+            }
+            catch(e)
+            {
+              console.error(
+                "rpc handler threw an error:" +
+                  " id=" + id +
+                  " result=" + JSON.stringify(result) +
+                  " ex=" + JSON.stringify(ex));
```

Comment:
Well it is used in a lot of places within the framework and in this file as well: https://github.com/derrell/qooxdoo/blob/27fea506e107f0564981a67bab7cd6254e5b12ed/framework/source/class/qx/io/remote/Rpc.js#L811